e2e/test/scenarios/admin-2/tenants.cy.spec.js

```diff
@@ -113,6 +113,14 @@ describe("Tenants - management", () => {
 
     H.modal().findByRole("textbox", { name: "User strategy" }).click();
     H.popover().findByText("Multi tenant").click();
+
+    cy.log("should show toast hint message about tenant collections");
+    cy.findAllByText(
+      "You can create tenant collections from the main Metabase navigation",
```

Comment:
Tbh I didn't even see this toast. It's too hidden, I'd expect this to scream at my face more IDK. That's UX question, not this PR's concern directly though.

Reply:
Good point, @AlessioLaiso we think the toast "You can create tenant collections from the main Metabase navigation" is a bit too subtle. Can we make that message pop somehow?